xml/cl.xml

```diff
@@ -1016,6 +1016,12 @@ server's OpenCL/api-docs repository.
         <enum bitpos="0"            name="CL_MEM_ALLOC_WRITE_COMBINED_INTEL"/>
     </enums>
 
+    <enums start="0x10000" end="0x1FFFF" name="cl_uint" vendor="Khronos" comment="cl_khronos_vendor_id">
+        <unused start="0x10000" end="0x10003" comment="These Khronos vendor IDs are already assigned in vk.xml"/>
+        <enum value="0x10004" name="CL_KHRONOS_VENDOR_ID_CODEPLAY" comment="Codeplay Software Ltd. Matches Khronos Vendor ID from vk.xml"/>
+        <unused start="0x10005" end="0x1FFFF" comment="This is the next unused available Khronos vendor ID"/>
```

Comment:
Per previous comment, I'd like to see one of two things happen: alter the process to require reservation in vk.xml first in all cases, or split this into a range for shared Vulkan-OpenCL assignment (say, 0x10000 - 0x100FF) and a range for OpenCL-only assignment (say, 0x10100-0x101FF), with a comment noting that reservations in the former range must be coordinated with vk.xml and reservations in the latter range may not be usable in other Khronos APIs. Something like this:

```
    <enums start="0x10000" end="0x100FF" name="cl_uint" vendor="Khronos" comment="cl_khronos_vendor_id - shared Vulkan IDs which must be reserved first in Vulkan's vk.xml">
        <enum value="0x10004" name="CL_KHRONOS_VENDOR_ID_CODEPLAY" comment="Codeplay Software Ltd. Matches Khronos Vendor ID from vk.xml"/>
    </enums>
    <enums start="0x10100" end="0x101FF" name="cl_uint" vendor="Khronos" comment="cl_khronos_vendor_id - OpenCL-only IDs">
        <unused start="0x10100" end="0x101FF" comment="Vendor IDs in this range can be reserved here and are usable only with OpenCL implementations"/>
    </enums>
```
If neither of these things is done, eventually someone will reserve a CL vendor ID which conflicts with a Vulkan vendor ID for another vendor.